client/ayon_core/tools/sceneinventory/models/containers.py

```diff
@@ -293,20 +307,37 @@ def get_version_items(self, product_ids):
 
             def version_sorted(entity):
                 return entity["version"]
+            containers = self.get_containers()
+            for repre_id in representation_ids:
```

Comment:
This change makes it uber slow when there is a lot of loaded containers in the scene.

Reply:
Yes - we should really group by project first, then batch the queries that way. I'd say this is required to fix within this PR